docs/nodes/nodes-library/nodes/Medium/README.md

```diff
@@ -0,0 +1,38 @@
+---
+permalink: /nodes/n8n-nodes-base.medium
+---
+
+# Medium
+
+[Medium](https://www.medium.com/) is an online publishing platform.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Medium/README.md).
+:::
+
+## Basic Operations
+
+- Publication
+	- Get all publications
+- Post
+	- Create a post
+
+## Example Usage
+
+This workflow allows you to get all publications from Medium. You can also find the [workflow](https://n8n.io/workflows/593) on the website. This example usage workflow uses the following two nodes.
+
+- [Start](../../core-nodes/Start/README.md)
+- [Medium]()
+
+The final workflow should look like the following image.
+
+![A workflow with the Medium node](./workflow.png)
```

Comment:
Is it possible to follow the dimensions of the image here? https://docs.n8n.io/nodes/n8n-nodes-base.questDb/